test/fixtures/styles/with-styles/output.astro

```diff
@@ -5,3 +5,5 @@
     color: red;
   }
 </style>
+
+<style set:html={someCss}></style>
```

Comment:
Does this depend on your self-closing preference? Would be nice to do `<style set:html={value} />` if users prefer that. I know scripts don't do that because we get highlighting issues (even though the compiler supports it)

Reply:
At the moment no (and Prettier doesn't actually have self closing preferences 😅). I'm planning to update this in a separate PR so that the `set:html` and `set:text` attributes allow elements to self close even on elements that don't usually allow it

> I know scripts don't do that because we get highlighting issues (even though the compiler supports it)

What do you mean? Seems to work fine: 
<img width="238" alt="image" src="https://user-images.githubusercontent.com/3019731/193085419-2e418bfa-c833-4bd7-a36f-c52a3b8bd267.png">

Reply:
Nice, sounds like a perfect follow-up. 

I guess I was mistaken on the `<script />` issue then, so that's great!